infra/src/dsl/value-files-generators/local-setup.spec.ts

```diff
@@ -52,7 +52,7 @@ describe('Local setup', () => {
         '..',
         '..',
         '..',
-      )}/mountebank-imposter-config.json:/app/default.json bbyars/mountebank:2.8.1 start --configfile=/app/default.json`,
+      )}/mountebank-imposter-config.json:/app/default.json docker.io/bbyars/mountebank:2.8.1 start --configfile=/app/default.json`,
```

Comment:
should we maybe retag this and push to ecr shared?

Reply:
For local dev needs it might be too much trouble?